README.md

```diff
@@ -147,6 +147,14 @@ Our current coverage reports only support Vitest unit tests by running
 npm run coverage
 ```
 
+## Start vite server and run cypress
+
+To run the vite server and run the cypress application at the same time
+
+```bash
+npm run cy:e2e
```

Comment:
In a future commit, we could also update the table of available scripts to have this command.